app/models/book.rb

```diff
@@ -1,3 +1,7 @@
 class Book < ActiveRecord::Base
+  belongs_to :author
+
+  scope :recent_five, -> {order(published_at: :desc).first 5}
```

Comment:
We need to discuss code style for our team. And configure our editors to fix such problems.